file/errors.go

```diff
@@ -0,0 +1,21 @@
+package file
+
+import (
+	"errors"
+	"strings"
+)
+
+var (
+	errDBIsClosed       = errors.New("DB is closed")
+	errContextClosing   = errors.New("context closing")
+	errInvalidParameter = errors.New("invalid parameter provided")
+)
+
+func isClosingError(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	return strings.Contains(err.Error(), errDBIsClosed.Error()) ||
```

Comment:
too specific, let's put back in elrond-go

Reply:
done